apps/central-scan/frontend/src/app_root.tsx

```diff
@@ -476,19 +476,35 @@ export function AppRoot({
     }
   }, [electionJustLoaded, usbDrive.status]);
 
+  // We check configureMutation's status and stop configuration attempts after a failed attempt.
+  // This prevents the app from looping with continued failed configurations.
+  // If the user ejects the USB drive, we reset configureMutation's status so configuration
+  // may be attempted when a new or modified USB drive is inserted.
+  useEffect(() => {
+    if (
+      (configureMutation.isError || configureMutation?.data?.err) &&
```

Comment:
You're checking whether a function is truthy. Did you mean `configureMutation?.data?.err()`?

Reply:
Good catch, sorry about that. Updated

Reply:
[1871198](https://github.com/votingworks/vxsuite/pull/3616/commits/1871198de2506e418f2d90209f5e350af97132d4)